wire-schema/src/jvmMain/java/com/squareup/wire/schema/PruningRules.kt

```diff
@@ -226,6 +230,10 @@ class PruningRules private constructor(builder: Builder) {
       check(oldest == null || newest == null || oldest!! <= newest!!) {
         "expected oldest $oldest <= newest $newest"
       }
+      val conflictingRules = includes.intersect(excludes)
+      check(conflictingRules.isEmpty()) {
+        "same rule(s) defined in both includes and excludes: ${conflictingRules.joinToString()}"
```

Comment:
supernit: `Same`

Reply:
We don't capitalize the error message just above so I just did the _same_.